os/fs/smartfs/smartfs_smart.c

```diff
@@ -186,37 +186,48 @@ static int smartfs_open(FAR struct file *filep, const char *relpath, int oflags,
 	/* Get the mountpoint inode reference from the file structure and the
 	 * mountpoint private data from the inode structure
 	 */
-
 	inode = filep->f_inode;
 	fs = inode->i_private;
 
 	DEBUGASSERT(fs != NULL);
 
```

Comment:
Typo in commit message "openend" => "opened"

Reply:
Done